Providers/Extras/Scripts/omsconfig_logrotate.conf

```diff
@@ -4,3 +4,10 @@
     weekly
     compress
 }
```

Comment:
I am wondering who makes use of this script configuration file. Is this some plugin ?